Lib/ast.py

```diff
@@ -936,11 +946,7 @@ def visit_Constant(self, node):
         value = node.value
         if isinstance(value, tuple):
             with self.delimit("(", ")"):
-                if len(value) == 1:
-                    self._write_constant(value[0])
-                    self.write(",")
-                else:
-                    self.interleave(lambda: self.write(", "), self._write_constant, value)
+                self.items_view(self._write_constant, value)
```

Comment:
There are no test for these cases, aren't they?

Reply:
as I stated in the issue, no. We can't test constant tuples with the classic way so I am planning to add a test after #17377 or #17760 (they both add a way to test generated source with given node instead of ast checks).

Reply:
> as I stated in the issue, no

Then shouldn't we add that code once the test can be added?

Reply:
We already had some missing tests, including constant tuples.

Reply:
> We already had some missing tests, including constant tuples.

That does not justify adding even more missing test IMHO ;)

Reply:
> We already dont have tests for that, 

Yes, unfortunately it isn't :/ Lets wait one of the PRs I mentioned before and we can add tests for constant tuples.